Resources/Public/Javascript/PageView/SearchInDocument.js

```diff
@@ -43,17 +43,233 @@ function resetStart() {
+
+    // add query params to result link
+    for(var i = 0; i < queryParams.length; i++) {
+        link += queryParams[i] + '=' + queryParams[queryParams[i]] + '&';
+    }
+    link = link.slice(0, -1);
+    return link;
+}
+
+function getNavigationButtons(start, numFound) {
+    var buttons = "";
+
+    if (start > 0) {
+        buttons += '<input type="button" id="tx-dlf-search-in-document-button-previous" class="button-previous" onclick="previousResultPage();" value="' + $('#tx-dlf-search-in-document-label-previous').text() + '" />';
+    }
+
+    if (numFound > (start + 20)) {
+        buttons += '<input type="button" id="tx-dlf-search-in-document-button-next" class="button-next" onclick="nextResultPage();" value="' + $('#tx-dlf-search-in-document-label-next').text() + '" />';
+    }
+    return buttons;
+}
+
+function getCurrentPage() {
+    var page = 1;
+    var queryParams = getCurrentQueryParams(getBaseUrl(" "));
+
+    for(var i = 0; i < queryParams.length; i++) {
+        var queryParam = queryParams[i].split('=');
+
+        if(queryParam[0] === $("input[id='tx-dlf-search-in-document-page']").attr('name')) {
+            page = queryParam[1];
+        }
+    }
+
+    return page;
+}
+
+function addImageHighlight(data) {
+    var page = getCurrentPage();
+
+    data['documents'].forEach(function (element, i) {
+        if(element['page'] === page) {
+            if (element['highlight'].length > 0) {
+                if(tx_dlf_viewer.map != null) {
+                    tx_dlf_viewer.displayHighlightWord(encodeURIComponent(getHighlights(element['highlight'])));
+                } else {
+                    setTimeout(addImageHighlight, 500, data);
+                }
+            }
+            addHighlightEffect(element['highlight']);
+        }
+    });
+}
+
+function triggerSearchAfterHitLoad() {
+    var queryParams = getCurrentQueryParams(getBaseUrl(" "));
+    var searchedQueryParam = $("input[id='tx-dlf-search-in-document-highlight-word']").attr('name');
+
+    for(var i = 0; i < queryParams.length; i++) {
+        var queryParam = queryParams[i].split('=');
+
+        if(searchedQueryParam && queryParam[0].indexOf(searchedQueryParam) !== -1) {
+            $("input[id='tx-dlf-search-in-document-query']").val(decodeURIComponent(queryParam[1]));
+            $("#tx-dlf-search-in-document-form").submit();
+            break;
+        }
+    }
+}
+
 $(document).ready(function() {
     $("#tx-dlf-search-in-document-form").submit(function(event) {
         // Stop form from submitting normally
         event.preventDefault();
+
         $('#tx-dlf-search-in-document-loading').show();
         $('#tx-dlf-search-in-document-clearing').hide();
         $('#tx-dlf-search-in-document-button-next').hide();
         $('#tx-dlf-search-in-document-button-previous').hide();
         // Send the data using post
         $.post(
-            "/",
+            "https://sdvtypo3ddbzeitungsportaldev.slub-dresden.de/",
```

Comment:
No, not really ;-)

Reply:
Yeah, it was specific for DDB, I will revert it ;)